WiFi_Sit_N__Wheelie_receiver/WiFi_Sit_N__Wheelie_receiver.ino

```diff
@@ -0,0 +1,104 @@
+#include <SPI.h>
+#include <Adafruit_WINC1500.h>
+#include <Adafruit_WINC1500Udp.h>
+
+//Define w1 and w2 as analog output to control the BLDC
+const int w1 = 6;
+const int w2 = 14;
+
+// Define the WINC1500 board connections below.
+// If you're following the Adafruit WINC1500 board
+// guide you don't need to modify these:
+#define WINC_CS   31
+#define WINC_IRQ  23
+#define WINC_RST  7
+#define WINC_EN   5     // or, tie EN to VCC and comment this out
+// The SPI pins of the WINC1500 (SCK, MOSI, MISO) should be
+// connected to the hardware SPI port of the Arduino.
+
+// Setup the WINC1500 connection with the pins above and the default hardware SPI.
+Adafruit_WINC1500 WiFi(WINC_CS, WINC_IRQ, WINC_RST);
+
+//Adafruit_WINC1500 WiFi;
+int status = WL_IDLE_STATUS;
+char ssid[] = "SSID";  //  your network SSID (name)
+char pass[] = "Password!";       // your network password
+int keyIndex = 0;            // your network key Index number (needed only for WEP)
+
+unsigned int localPort = 2390;      // local port to listen on
+
+char packetBuffer[255]; //buffer to hold incoming packet
+
+Adafruit_WINC1500UDP Udp;
+
+void setup() {
+#ifdef WINC_EN
+  pinMode(WINC_EN, OUTPUT);
+  digitalWrite(WINC_EN, HIGH);
+#endif
+
+  //Initialize serial and wait for port to open:
+  Serial.begin(9600);
+
+  // check for the presence of the shield:
+  if (WiFi.status() == WL_NO_SHIELD) {
+    Serial.println("WiFi shield not present");
+    // don't continue:
+    while (true);
+  }
+
+  // attempt to connect to Wifi network:
+  while ( status != WL_CONNECTED) {
+    Serial.print("Attempting to connect to SSID: ");
+    Serial.println(ssid);
+    // Connect to WPA/WPA2 network. Change this line if using open or WEP network:
+    status = WiFi.begin(ssid, pass);
+
+    // wait 5 seconds for connection:
+    delay(5000);
+  }
+  Serial.println("Connected to wifi");
+  printWifiStatus();
+
+  Serial.println("\nStarting connection to server...");
+  // if you get a connection, report back via serial:
+  Udp.begin(localPort);
+}
+
+void loop() {
+
+  // if there's data available, read a packet
+  int packetSize = Udp.parsePacket();
+
+  if (packetSize)
+  {
+    int len = Udp.read(packetBuffer,255);
+    if (len > 0) packetBuffer[len] = 0;
+    Serial.print("RX: ");
```

Comment:
So this is just dumping the information from the UDP to the Serial monitor right?